tide_core.install

```diff
@@ -1548,3 +1548,36 @@ function tide_core_update_8064() {
     }
   }
 }
+
+/**
+ * Add contact field to user account(business name, phone, notes).
+ */
+function tide_core_update_8065() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_core') . '/config/install'];
+  $configs = [
+    'field.storage.user.field_business_contact_number' => 'field_storage_config',
+    'field.storage.user.field_business_name' => 'field_storage_config',
+    'field.storage.user.field_notes' => 'field_storage_config',
+  ];
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    $id = $storage->getIDFromConfigName($config, $storage->getEntityType()->getConfigPrefix());
+    if ($storage->load($id) == NULL) {
+      $config_entity = $storage->createFromStorageRecord($config_read);
+      $config_entity->save();
+    }
+  }
+}
+
+/**
+ * Update user form display and authenticated user permission.
+ */
+function tide_core_update_8066() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_core') . '/config/install'];
+  _tide_import_single_config('user.role.authenticated', $config_location);
+  _tide_import_single_config('user.role.site_admin', $config_location);
```

Comment:
does the site_admin needs all these permissions though ! isn't it like a normal other fields that just shows up and users can input things. I don't think it needs the field level permission. Can you please check if it works without these field level permission and user can input the values.